evm_loader/lib/src/config.rs

```diff
@@ -21,67 +16,13 @@ pub struct Config {
     pub keypair_path: String,
 }
 
-impl Config {
-    pub fn build_solana_rpc_client(&self) -> RpcClient {
-        RpcClient::new_with_commitment(self.json_rpc_url.clone(), self.commitment)
-    }
-
-    pub fn build_clone_solana_rpc_client(&self) -> CloneRpcClient {
-        CloneRpcClient::new(self.build_solana_rpc_client())
-    }
-}
-
-/// # Errors
-pub fn create_from_api_config(api_config: &APIOptions) -> Result<Config, NeonError> {
-    let solana_cli_config: SolanaConfig =
-        if let Some(path) = api_config.solana_cli_config_path.clone() {
-            solana_cli_config::Config::load(path.as_str()).unwrap_or_default()
-        } else {
-            solana_cli_config::Config::default()
-        };
-
-    let commitment = CommitmentConfig::from_str(&api_config.commitment)
-        .unwrap_or_else(|_| CommitmentConfig::confirmed());
-
-    let json_rpc_url = normalize_to_url_if_moniker(api_config.json_rpc_url.clone());
-
-    let evm_loader: Pubkey = if let Ok(val) = Pubkey::from_str(&api_config.evm_loader) {
-        val
-    } else {
-        return Err(NeonError::EvmLoaderNotSpecified);
-    };
-
-    let keypair_path: String = api_config.keypair.clone();
-
-    let fee_payer = keypair_from_path(
-        &Default::default(),
-        &api_config.fee_payer,
-        "fee_payer",
-        true,
-    )
-    .ok();
-
-    let db_config: Option<ChDbConfig> = Option::from(api_config.db_config.clone());
-
-    Ok(Config {
-        evm_loader,
-        fee_payer,
-        commitment,
-        solana_cli_config,
-        db_config,
-        json_rpc_url,
-        keypair_path,
-    })
-}
-
 #[derive(Debug, Deserialize, Serialize)]
 pub struct APIOptions {
     pub solana_cli_config_path: Option<String>,
-    pub commitment: String,
+    pub commitment: CommitmentConfig,
     pub json_rpc_url: String,
-    pub evm_loader: String,
-    pub keypair: String,
-    pub fee_payer: String,
+    pub evm_loader: Pubkey,
+    pub operator: Pubkey,
```

Comment:
I suggest replacing all `operator` names in this PR with `solana_key_for_config` or maybe something like `tx_simulation_payer`, to emphasize that this pubkey does not actually pay anything, it is only used for tx simulation.

We should avoid using `operator` again, as it can be confused with Neon operator pubkey.

Reply:
Done